### ISSUE TITLE
Accpet full URL in osc.open()

### ISSUE THREAD

**Issue by arnoson** [adzialocha/osc-js#44]
It would be nice to shorten this:
```js
osc.open({ host: '127.0.0.1', port: 80, secure: true  })
```
to something like this:
```js
osc.open('wss://127.0.0.1:80')
```
If you are also interested in this I can implement this feature and create a pull request (:

**Comment by adzialocha** [comment]
I agree, thats definitely useful @arnoson, it would be great if you could write a PR for this :smiley_cat:
